eth/protocols/eth/receipt.go

```diff
@@ -337,6 +337,15 @@ func (rl *ReceiptList68) EncodeRLP(_w io.Writer) error {
 	return w.Flush()
 }
 
+// ExcludeStateSync removes the state sync transaction from the list. We use
+// the property of zero gas used to identify a state-sync transaction in the
+// list.
+func (rl *ReceiptList68) ExcludeStateSync() {
+	if len(rl.items) > 0 && rl.items[len(rl.items)-1].GasUsed == 0 {
```

Comment:
Here `GasUsed` is used, which isn't consistent with other places where `CumulativeGasUsed`.